examples/graph_gps.py

```diff
@@ -65,8 +79,33 @@ def forward(self, x, pe, edge_index, edge_attr, batch):
         return self.mlp(x)
 
 
+class RedrawProjection(torch.nn.Module):
+    def __init__(self, model: torch.nn.Module,
+                 redraw_interval: Optional[int] = None):
```

Comment:
Why allow `redraw_interval` to be `None`?

Reply:
Here we set the `redraw_interval` to `None` when we use multihead attention in this example as we only do the redraw for performer.